packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -14,7 +14,7 @@ class BuildInfo {
   const BuildInfo(
     this.mode,
     this.flavor, {
-    this.trackWidgetCreation = false,
+    this.trackWidgetCreation = true,
```

Comment:
What's the reason for flipping the default? Can this just be required so that we're sure no call sites get flipped by accident?

Reply:
Switched it back, this was unrelated